libs/iblprefilter/src/materials/iblprefilter.mat

```diff
@@ -0,0 +1,148 @@
+material {
+    name : iblprefilter,
+    parameters : [
+        {
+            type : samplerCubemap,
+            name : environment,
+            precision: medium
+        },
+        {
+            type : sampler2d,
+            name : kernel,
+            precision: high
+        },
+        {
+            type : float2,
+            name : compress,
+            precision: medium
+        },
+        {
+            type : float,
+            name : side,
+            precision: medium
+        },
+        {
+            type : float,
+            name : invWeightSum,
+            precision: high
+        },
+        {
+            type : uint,
+            name : sampleCount,
+            precision: medium
+        }
+    ],
+    outputs : [
+        {
+            name : outx,
+            target : color,
+            type : float4
+        },
+        {
+            name : outy,
+            target : color,
+            type : float4
+        },
+        {
+            name : outz,
+            target : color,
+            type : float4
+        }
+    ],
+    variables : [
+        vertex
+    ],
+    domain : postprocess,
+    depthWrite : false,
+    depthCulling : false
+}
+
+vertex {
+    void postProcessVertex(inout PostProcessVertexInputs postProcess) {
+        postProcess.vertex.xy = postProcess.normalizedUV;
+    }
+}
+
+fragment {
+
+void dummy() {}
```

Comment:
what's this?

Reply:
oh it's just to thwart CLion -- it doesn't like our jsonish meta data stuff at the top of the materials.